docs/hardware/troubleshooting.md

```diff
@@ -0,0 +1,55 @@
+## **TITLE: TROUBLESHOOTING MANUAL**
+
+### **POSSIBLE ERRORS**
+
+1. Code not compiling. 
+2. Incorrect voltage displaying as compared to a TRMS (True Root Mean Square)
+meter. 
+3. Incorrect current/power display. 
+4. Meter refusing to send data to things board or not connecting to API
+(Application Programming Interface). 
+5. Keypad not displaying what is pressed correctly. 
+6. Incorrect LED (Light Emitting Diode) display
+
+### **SOLUTION TO POSSIBLE ERROR**
+
+&nbsp; &nbsp; &nbsp;  &nbsp; &nbsp; &nbsp; 1. CODE NOT COMPILING: <br>
+Code may not compile due to library used in code not installed in the library folder of
+the Arduino IDE (Integrated Development Environment) installation path. Developer
+should copy the necessary library to the installation path, and restart the IDEthentryre-compiling or install the libraries from
+
+<span style="background: yellow;">**SKETCH** </span>  → <span style="background: yellow;">**INCLUDE LIBRARY** </span> → <span style="background: yellow;">**ADD ZIP LIBRARY** </span>, then add  the necessary library with the complain  on the **verbose** output.
```

Comment:
Instead of using `<span style="background: yellow;"> ... </span>` can you use "Mark" tags? https://enaccess.github.io/OpenSmartMeter/example/#mark

So, for example, `==**SKETCH**==`